azure-resources/Compute/virtualMachines/kql/302fda08-ee65-4fbe-a916-6dc0b33169c4.kql

```diff
@@ -0,0 +1,7 @@
+// Azure Resource Graph Query
+// Find all Virtual Machines not associated with a Capacity Reservation, and provide details for Capacity Reservation like vmSize, location, and zone.
+resources
+| where type =~ 'Microsoft.Compute/virtualMachines'
+| where isnull( properties.capacityReservation)
+| extend zoneValue = iff((isnull(zones)), "null", zones)
+| project recommendationId="302fda08-ee65-4fbe-a916-6dc0b33169c4", name, id, tags, param1= strcat("vmSize: ", properties.hardwareProfile.vmSize), param2 = location, param3 = strcat("Zone: ", zoneValue)
```

Comment:
- Added whitespaces to the side of operators.
- Did PascalCase the param1 label because the param3 label used PascalCase.
- Added a label to the param2 because other params have labels.

```suggestion
| project recommendationId = "302fda08-ee65-4fbe-a916-6dc0b33169c4", name, id, tags, param1 = strcat("VmSize: ", properties.hardwareProfile.vmSize), param2 = strcat("Location: ", location), param3 = strcat("Zone: ", zoneValue)
```